pom.xml

```diff
@@ -686,6 +686,77 @@
       </properties>
     </profile>
 
+    <profile>
+      <id>testCoverage</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+        <property>
+          <name>testCoverage</name>
+        </property>
+      </activation>
+      <properties>
+        <test.runner.jvm.settings.additional>@{argLine}</test.runner.jvm.settings.additional>
+      </properties>
+        <build>
+            <plugins>
+                <plugin>
+                    <groupId>org.jacoco</groupId>
+                    <artifactId>jacoco-maven-plugin</artifactId>
+                    <version>0.7.9</version>
+                    <executions>
+                        <execution>
+                            <id>default-prepare-agent</id>
+                            <goals>
+                                <goal>prepare-agent</goal>
+                            </goals>
+                        </execution>
+                        <execution>
+                            <id>default-prepare-agent-integration</id>
+                            <goals>
+                                <goal>prepare-agent-integration</goal>
+                            </goals>
+                        </execution>
+                        <execution>
+                            <id>merge</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>merge</goal>
+                            </goals>
+                            <configuration>
+                                <destFile>${project.build.directory}/jacoco-aggregated.exec</destFile>
```

Comment:
Should `jacoco-aggregated.exec` be added to `.gitignore`?

Reply:
project build directory is a target directory and those are ignored anyway already